plugins/ci/skills/analyze-disruption/test_find_disruption_runs.py

```
@@ -31,6 +32,119 @@ def test_parse_grafana_url_extracts_vars():
+    from contextlib import redirect_stderr
+
+    buf = io.StringIO()
+    try:
+        with redirect_stderr(buf):
+            main([
+                "--grafana-url",
+                "https://grafana-loki.ci.openshift.org/d/abc/dash"
+                "?var-backend=host-to-host-new-connections&var-backend=kube-api-new-connections"
+                "&var-releases=5.0",
+            ])
+        assert False, "should have exited"
```

Ruff (0.16.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)